jobs/ci-kubernetes-e2e-gke-cvm-1-5-cvm-1-7-upgrade-cluster.env

```diff
@@ -6,5 +6,10 @@ KUBE_GKE_IMAGE_TYPE=container_vm
 
 PROJECT=gke-up-c1-3-clat-up-clu
 
+localSkip1="Daemon\ set\ \[Serial\]\ should\ run\ and\ stop\ complex\ \daemon\ with\ node\ affinity"
+localSkip2="\[SchedulerPredicates\]"
+localSkip3="\[SchedulerPriorities\]"
+
+GINKGO_TEST_ARGS=--ginkgo.skip="${localSkip1}|${localSkip2}|${localSkip3}"
```

Comment:
it might also be a good idea to have a comment linking to the issues here; that'll be easier than git blame when trying to figure out why these are disabled.